test/parallel/parallel.status

```diff
@@ -97,3 +99,7 @@ test-http-pipeline-flood: SKIP
 [$asan==on]
 # https://github.com/nodejs/node/issues/39655
 test-cluster-primary-error: PASS, FLAKY
+
+[$arch==s390x]
+# https://github.com/nodejs/node/issues/43465
+test-http-server-request-timeouts-mixed: PASS, FLAKY
```

Comment:
This isn't needed since the test is also being marked flaky under `$system==linux` above.

Reply:
Thank you. Force pushed.